generator/tests/testdata/golden/php/default/endpoints_google_owned.golden

```diff
@@ -52,6 +53,7 @@ class EccoDomaniIeri extends \Google\Service
   {
     parent::__construct($clientOrConfig);
     $this->rootUrl = $rootUrl ?: 'https://peer-pontus.appspot.com/_ah/api/';
```

Comment:
Shouldn't this be removed?

Reply:
For backwards compatibility it's better to deprecate it than remove it. It's possible for a previous version of `google/apiclient` to install a _newer_ version of this library (`google/apiclient-services`), at which point this would break if removed.